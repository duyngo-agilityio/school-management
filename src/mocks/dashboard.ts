export const CONTENT_DASHBOARD = [
  {
    icon: '',
    title: 'Add other admins',
    description: 'Create rich course content and coaching products for your students. When you give them a pricing plan, they’ll appear on your site!'
  },
  {
    icon: '',
    title: 'Add classes',
    description: 'Create rich course content and coaching products for your students. When you give them a pricing plan, they’ll appear on your site!'
  },
  {
    icon: '',
    title: 'Add students',
    description: 'Create rich course content and coaching products for your students. When you give them a pricing plan, they’ll appear on your site!'
  },
];
