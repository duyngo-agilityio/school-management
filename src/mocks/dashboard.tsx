// Icons
import { BankIcon, ProfileAddIcon, TeacherIcon } from '@/icons';

export const CONTENT_DASHBOARD = [
  {
    id: 1,
    icon: <ProfileAddIcon />,
    title: 'Add other admins',
    description:
      'Create rich course content and coaching products for your students. When you give them a pricing plan, they’ll appear on your site!',
  },
  {
    id: 2,
    icon: <BankIcon />,
    title: 'Add classes',
    description:
      'Create rich course content and coaching products for your students. When you give them a pricing plan, they’ll appear on your site!',
  },
  {
    id: 3,
    icon: <TeacherIcon />,
    title: 'Add students',
    description:
      'Create rich course content and coaching products for your students. When you give them a pricing plan, they’ll appear on your site!',
  },
];
