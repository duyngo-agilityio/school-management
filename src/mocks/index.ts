export * from './students';
export * from './teachers';
export * from './dashboard';
