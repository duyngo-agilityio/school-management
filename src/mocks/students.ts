export const MOCK_STUDENTS = [
  {
    fullName: "Stacy Prohaska",
    email: "Isaiah16@gmail.com",
    gender: 0,
    age: 16,
    description: "Error eaque doloribus maiores repudiandae. Doloribus iure fuga blanditiis saepe quibusdam. Quasi suscipit ratione quod. Rem tenetur assumenda tenetur aliquam autem vel nostrum. Laboriosam laborum inventore illo nesciunt vel vel soluta dignissimos ipsam. Molestiae ea labore totam enim.",
    phoneNumber: "488-713-9023",
    avatar: "https://cloudflare-ipfs.com/ipfs/Qmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye/avatar/922.jpg",
    id: "6"
    },
    {
    fullName: "Victoria Runolfsson",
    email: "Cortez36@hotmail.com",
    gender: 0,
    age: 16,
    description: "Quia architecto nesciunt impedit vel minus asperiores ad ex. Repudiandae ex enim repellat reiciendis modi laudantium. Sint voluptatum reprehenderit suscipit ea tempora.",
    phoneNumber: "446-814-2948",
    avatar: "https://cloudflare-ipfs.com/ipfs/Qmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye/avatar/1233.jpg",
    id: "7"
    },
    {
    fullName: "Courtney Stroman",
    email: "Tyrese_Hahn@gmail.com",
    gender: 0,
    age: 16,
    description: "Et placeat harum expedita eveniet cumque. Provident doloribus esse perferendis error ullam facere molestias eius. Magnam incidunt quasi facilis tempore aperiam sapiente. Voluptate explicabo itaque a quasi aliquid. Autem magnam ipsum eius voluptatum commodi.",
    phoneNumber: "244-577-1884",
    avatar: "https://cloudflare-ipfs.com/ipfs/Qmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye/avatar/267.jpg",
    id: "8"
    },
    {
    fullName: "Daisy Schaden",
    email: "Jude_Schaefer54@gmail.com",
    gender: 1,
    age: 16,
    description: "Unde occaecati autem corporis sit officiis impedit quidem iste. Nulla molestias reprehenderit repudiandae. Sunt suscipit cupiditate expedita ducimus minus eos aliquid. Omnis itaque ea deleniti corrupti corporis enim quibusdam. Corrupti fugiat alias voluptas dolorum mollitia odit temporibus. Quam optio architecto magni dolore assumenda expedita voluptate culpa.",
    phoneNumber: "473-516-5777",
    avatar: "https://cloudflare-ipfs.com/ipfs/Qmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye/avatar/458.jpg",
    id: "9"
    },
    {
    fullName: "Michelle Cormier",
    email: "Courtney_Legros@gmail.com",
    gender: 1,
    age: 16,
    description: "Voluptate voluptates sint quidem modi officia ea. Quisquam tenetur doloribus natus vitae ea est excepturi. Quaerat dolorem expedita itaque sit veniam pariatur laudantium deserunt. Temporibus odit mollitia alias. Asperiores temporibus ratione mollitia cumque.",
    phoneNumber: "380-976-9633",
    avatar: "https://cloudflare-ipfs.com/ipfs/Qmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye/avatar/427.jpg",
    id: "10"
    },
];
