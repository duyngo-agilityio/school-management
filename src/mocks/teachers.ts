export const MOCK_TEACHERS = [{
  fullName: "Naomi Murazik",
  email: "Therese_Johnston16@hotmail.com",
  gender: 1,
  age: 43,
  description: "Tenetur nemo ab mollitia sapiente odit deserunt. Veritatis autem nihil nisi praesentium eaque quos ab doloremque. Odit et quaerat excepturi unde. Aperiam suscipit enim cumque voluptate.",
  phoneNumber: "355-641-5320",
  avatar: "https://cloudflare-ipfs.com/ipfs/Qmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye/avatar/243.jpg",
  subject: "Literature",
  id: "1"
  },
  {
  fullName: "Jan Williamson DDS",
  email: "Lavinia.Bahringer16@hotmail.com",
  gender: 0,
  age: 30,
  description: "Vitae ipsum repellat natus magnam iusto dolorum optio unde. In cumque eveniet non excepturi. Deleniti tempore illum expedita.",
  phoneNumber: "264-264-2834",
  avatar: "https://cloudflare-ipfs.com/ipfs/Qmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye/avatar/610.jpg",
  subject: "Physics",
  id: "2"
  },
  {
  fullName: "Javier Rice Sr.",
  email: "Lue.Fay90@hotmail.com",
  gender: 1,
  age: 35,
  description: "Illum sapiente explicabo perferendis architecto. Maiores eos fuga nihil exercitationem. Consequatur id ex laudantium iste repudiandae accusamus officiis hic.",
  phoneNumber: "509-374-7387",
  avatar: "https://cloudflare-ipfs.com/ipfs/Qmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye/avatar/588.jpg",
  subject: "Math",
  id: "3"
  },
  {
  fullName: "Bonnie Weissnat",
  email: "Jake_Schamberger@gmail.com",
  gender: 0,
  age: 40,
  description: "Numquam rerum itaque. Vel accusantium eligendi magni architecto nisi molestiae rerum corporis debitis. Pariatur placeat ipsum earum. Porro nobis molestias odio nemo autem eligendi non. Tenetur sint maiores. Voluptas fugit ipsam id suscipit ipsam aut ea animi.",
  phoneNumber: "706-895-6640",
  avatar: "https://cloudflare-ipfs.com/ipfs/Qmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye/avatar/289.jpg",
  subject: "Biology",
  id: "4"
  },
  {
  fullName: "Shannon Conroy",
  email: "Isom.Hettinger95@gmail.com",
  gender: 0,
  age: 38,
  description: "Molestiae veniam quasi consectetur officiis commodi tempora. Nesciunt ut autem dolor provident. Accusamus explicabo veritatis eos alias eum modi labore. Adipisci aperiam voluptas voluptas natus id consectetur ducimus.",
  phoneNumber: "494-572-1501",
  avatar: "https://cloudflare-ipfs.com/ipfs/Qmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye/avatar/100.jpg",
  subject: "Math",
  id: "5"
  },
];
