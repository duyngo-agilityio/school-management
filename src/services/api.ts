type HttpMethod = 'GET' | 'POST' | 'PUT' | 'DELETE';

export const apiRequest = async <T>({
  endpoint,
  method,
  payload,
  configOptions,
}: {
  endpoint: string;
  method: HttpMethod;
  payload?: T;
  configOptions?: RequestInit;
}) => {
  const requestOptions: RequestInit = {
    method,
    headers: {
      'Content-Type': 'application/json',
    },
    ...configOptions,
  };

  if (method === 'POST' || method === 'PUT') {
    requestOptions.body = JSON.stringify(payload);
  }

  const response = await fetch(endpoint, requestOptions);
  const data = await response.json();

  if (!response.ok) {
    throw new Error(data?.error?.message);
  }

  return data;
};
