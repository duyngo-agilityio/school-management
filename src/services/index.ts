export * from './student';
export * from './teacher';
export * from './api';
