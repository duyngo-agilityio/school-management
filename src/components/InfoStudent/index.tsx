// Utils
import { createIcons } from '@/utils';

// Components
import { Box, Flex, Text } from '@chakra-ui/react';
import Icon from '../Icon';
import Profile from '../Profile';

// Services
import { getStudentById } from '@/services';
import { notFound } from 'next/navigation';

const InfoStudent = async ({ params }: { params: { id: string } }) => {
  const data = await getStudentById(params.id);
  const { age, fullName, gender, id, avatar, description } = data;

  if (!data) notFound();

  return (
    <Flex flexDirection="column" alignItems="center" px="39px" mt="45px">
      <Profile
        id={id}
        title={fullName}
        src={avatar}
        variant="md"
        subTitle="Physical Student"
      />
      <Flex flexDirection="row" mt="37px">
        {createIcons('24', '24').map(({ id, icon }) => (
          <Flex
            key={id}
            alignItems="center"
            justifyContent="center"
            mx="13px"
            w="36px"
            h="36px"
            bg="backgroundLinkWater"
            borderRadius="lg"
          >
            <Icon icon={icon} />
          </Flex>
        ))}
      </Flex>
      <Flex flexDirection="column" mt="47px">
        <Box>
          <Text variant="title" fontWeight="semibold" size="md">
            About
          </Text>
          <Text variant="subTitle" fontSize="md" textAlign="justify" mt="16px">
            {description}
          </Text>
        </Box>
        <Flex mt="30px" gap="150">
          <Box>
            <Text variant="title" fontWeight="semibold" size="md">
              Age
            </Text>
            <Text variant="subTitle" fontSize="md" mt="10px">
              {age}
            </Text>
          </Box>
          <Box>
            <Text variant="title" fontWeight="semibold" size="md">
              Gender
            </Text>
            <Text variant="subTitle" fontSize="md" mt="10px">
              {gender}
            </Text>
          </Box>
        </Flex>
      </Flex>
    </Flex>
  );
};

export default InfoStudent;
