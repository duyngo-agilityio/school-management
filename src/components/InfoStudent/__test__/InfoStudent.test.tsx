import { MOCK_STUDENTS } from '@/mocks';
import InfoStudent from '..';
import { render, waitFor } from '@testing-library/react';
import { getStudentById } from '@/services';

jest.mock('next/navigation', () => ({
  notFound: jest.fn().mockReturnValue('not found'),
}));

jest.mock('@/services', () => ({
  getStudentById: jest.fn(),
}));

describe('InfoStudent', () => {
  it('match snapshot', async () => {
    (getStudentById as jest.Mock).mockResolvedValue(MOCK_STUDENTS[1]);

    const { container } = render(
      await InfoStudent({
        params: {
          id: '7',
        },
      }),
    );

    await waitFor(() => {
      expect(container).toMatchSnapshot();
    });
  });

  it('match snapshot with not found', async () => {
    (getStudentById as jest.Mock).mockResolvedValue({});

    const { container } = render(
      await InfoStudent({
        params: {
          id: '7',
        },
      }),
    );

    await waitFor(() => {
      expect(container).toMatchSnapshot();
    });
  });
});
