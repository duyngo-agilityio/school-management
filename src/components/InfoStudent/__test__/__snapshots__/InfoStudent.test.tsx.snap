// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoStudent match snapshot 1`] = `
<div>
  <div
    class="css-1pduq2g"
  >
    <div
      class="css-0"
    >
      <div
        class="css-s1wtk5"
      >
        <p
          class="chakra-text css-0"
        >
          7
        </p>
        <div
          class="css-79elbk"
          style="width: 180px; height: 180px;"
        >
          <img
            alt="Victoria Runolfsson"
            data-nimg="fill"
            decoding="async"
            loading="lazy"
            sizes="100vw"
            src="/_next/image?url=https%3A%2F%2Fcloudflare-ipfs.com%2Fipfs%2FQmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye%2Favatar%2F1233.jpg&w=3840&q=75"
            srcset="/_next/image?url=https%3A%2F%2Fcloudflare-ipfs.com%2Fipfs%2FQmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye%2Favatar%2F1233.jpg&w=640&q=75 640w, /_next/image?url=https%3A%2F%2Fcloudflare-ipfs.com%2Fipfs%2FQmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye%2Favatar%2F1233.jpg&w=750&q=75 750w, /_next/image?url=https%3A%2F%2Fcloudflare-ipfs.com%2Fipfs%2FQmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye%2Favatar%2F1233.jpg&w=828&q=75 828w, /_next/image?url=https%3A%2F%2Fcloudflare-ipfs.com%2Fipfs%2FQmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye%2Favatar%2F1233.jpg&w=1080&q=75 1080w, /_next/image?url=https%3A%2F%2Fcloudflare-ipfs.com%2Fipfs%2FQmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye%2Favatar%2F1233.jpg&w=1200&q=75 1200w, /_next/image?url=https%3A%2F%2Fcloudflare-ipfs.com%2Fipfs%2FQmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye%2Favatar%2F1233.jpg&w=1920&q=75 1920w, /_next/image?url=https%3A%2F%2Fcloudflare-ipfs.com%2Fipfs%2FQmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye%2Favatar%2F1233.jpg&w=2048&q=75 2048w, /_next/image?url=https%3A%2F%2Fcloudflare-ipfs.com%2Fipfs%2FQmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye%2Favatar%2F1233.jpg&w=3840&q=75 3840w"
            style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent; border-radius: 50%; background-size: cover; background-position: 50% 50%; background-repeat: no-repeat;"
          />
        </div>
      </div>
      <div
        class="css-izuqqr"
      >
        <p
          class="chakra-text css-0"
        >
          Victoria Runolfsson
        </p>
        <p
          class="chakra-text css-0"
        >
          Physical Student
        </p>
      </div>
    </div>
    <div
      class="css-q344ct"
    >
      <div
        class="css-h2ayij"
      >
        <div
          class="css-h5bbgz"
        >
          <svg
            fill="none"
            height="24"
            viewBox="0 0 24 24"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M10.05 2.52997L4.03002 6.45997C2.10002 7.71997 2.10002 10.54 4.03002 11.8L10.05 15.73C11.13 16.44 12.91 16.44 13.99 15.73L19.98 11.8C21.9 10.54 21.9 7.72997 19.98 6.46997L13.99 2.53997C12.91 1.81997 11.13 1.81997 10.05 2.52997Z"
              stroke="#A7A7A7"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="1.5"
            />
            <path
              d="M5.63012 13.08L5.62012 17.77C5.62012 19.04 6.60012 20.4 7.80012 20.8L10.9901 21.86C11.5401 22.04 12.4501 22.04 13.0101 21.86L16.2001 20.8C17.4001 20.4 18.3801 19.04 18.3801 17.77V13.13"
              stroke="#A7A7A7"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="1.5"
            />
            <path
              d="M21.3999 15V9"
              stroke="#A7A7A7"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="1.5"
            />
          </svg>
        </div>
      </div>
      <div
        class="css-h2ayij"
      >
        <div
          class="css-h5bbgz"
        >
          <svg
            fill="none"
            height="24"
            viewBox="0 0 30 30"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M27.4625 22.9125C27.4625 23.3625 27.3625 23.825 27.15 24.275C26.9375 24.725 26.6625 25.15 26.3 25.55C25.6875 26.225 25.0125 26.7125 24.25 27.025C23.5 27.3375 22.6875 27.5 21.8125 27.5C20.5375 27.5 19.175 27.2 17.7375 26.5875C16.3 25.975 14.8625 25.15 13.4375 24.1125C12 23.0625 10.6375 21.9 9.3375 20.6125C8.05 19.3125 6.8875 17.95 5.85 16.525C4.825 15.1 4 13.675 3.4 12.2625C2.8 10.8375 2.5 9.475 2.5 8.175C2.5 7.325 2.65 6.5125 2.95 5.7625C3.25 5 3.725 4.3 4.3875 3.675C5.1875 2.8875 6.0625 2.5 6.9875 2.5C7.3375 2.5 7.6875 2.575 8 2.725C8.325 2.875 8.6125 3.1 8.8375 3.425L11.7375 7.5125C11.9625 7.825 12.125 8.1125 12.2375 8.3875C12.35 8.65 12.4125 8.9125 12.4125 9.15C12.4125 9.45 12.325 9.75 12.15 10.0375C11.9875 10.325 11.75 10.625 11.45 10.925L10.5 11.9125C10.3625 12.05 10.3 12.2125 10.3 12.4125C10.3 12.5125 10.3125 12.6 10.3375 12.7C10.375 12.8 10.4125 12.875 10.4375 12.95C10.6625 13.3625 11.05 13.9 11.6 14.55C12.1625 15.2 12.7625 15.8625 13.4125 16.525C14.0875 17.1875 14.7375 17.8 15.4 18.3625C16.05 18.9125 16.5875 19.2875 17.0125 19.5125C17.075 19.5375 17.15 19.575 17.2375 19.6125C17.3375 19.65 17.4375 19.6625 17.55 19.6625C17.7625 19.6625 17.925 19.5875 18.0625 19.45L19.0125 18.5125C19.325 18.2 19.625 17.9625 19.9125 17.8125C20.2 17.6375 20.4875 17.55 20.8 17.55C21.0375 17.55 21.2875 17.6 21.5625 17.7125C21.8375 17.825 22.125 17.9875 22.4375 18.2L26.575 21.1375C26.9 21.3625 27.125 21.625 27.2625 21.9375C27.3875 22.25 27.4625 22.5625 27.4625 22.9125Z"
              stroke="#A7A7A7"
              stroke-miterlimit="10"
              stroke-width="1.5"
            />
            <path
              d="M23.125 11.25C23.125 10.5 22.5375 9.35 21.6625 8.4125C20.8625 7.55 19.8 6.875 18.75 6.875"
              stroke="#A7A7A7"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="1.5"
            />
            <path
              d="M27.5 11.25C27.5 6.4125 23.5875 2.5 18.75 2.5"
              stroke="#A7A7A7"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="1.5"
            />
          </svg>
        </div>
      </div>
      <div
        class="css-h2ayij"
      >
        <div
          class="css-h5bbgz"
        >
          <svg
            fill="none"
            height="24"
            viewBox="0 0 30 30"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M21.25 25.625H8.75C5 25.625 2.5 23.75 2.5 19.375V10.625C2.5 6.25 5 4.375 8.75 4.375H21.25C25 4.375 27.5 6.25 27.5 10.625V19.375C27.5 23.75 25 25.625 21.25 25.625Z"
              stroke="#A7A7A7"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-miterlimit="10"
              stroke-width="1.5"
            />
            <path
              d="M21.25 11.25L17.3375 14.375C16.05 15.4 13.9375 15.4 12.65 14.375L8.75 11.25"
              stroke="#A7A7A7"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-miterlimit="10"
              stroke-width="1.5"
            />
          </svg>
        </div>
      </div>
    </div>
    <div
      class="css-1cruzfr"
    >
      <div
        class="css-0"
      >
        <p
          class="chakra-text css-10ircq3"
        >
          About
        </p>
        <p
          class="chakra-text css-1nykzj4"
        >
          Quia architecto nesciunt impedit vel minus asperiores ad ex. Repudiandae ex enim repellat reiciendis modi laudantium. Sint voluptatum reprehenderit suscipit ea tempora.
        </p>
      </div>
      <div
        class="css-1409d40"
      >
        <div
          class="css-0"
        >
          <p
            class="chakra-text css-10ircq3"
          >
            Age
          </p>
          <p
            class="chakra-text css-evennk"
          >
            16
          </p>
        </div>
        <div
          class="css-0"
        >
          <p
            class="chakra-text css-10ircq3"
          >
            Gender
          </p>
          <p
            class="chakra-text css-evennk"
          >
            0
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`InfoStudent match snapshot with not found 1`] = `
<div>
  <div
    class="css-1pduq2g"
  >
    <div
      class="css-0"
    >
      <div
        class="css-79elbk"
        style="width: 180px; height: 180px;"
      >
        <img
          data-nimg="fill"
          decoding="async"
          src=""
          style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent; border-radius: 50%; background-size: cover; background-position: 50% 50%; background-repeat: no-repeat;"
        />
      </div>
      <div
        class="css-izuqqr"
      >
        <p
          class="chakra-text css-0"
        />
        <p
          class="chakra-text css-0"
        >
          Physical Student
        </p>
      </div>
    </div>
    <div
      class="css-q344ct"
    >
      <div
        class="css-h2ayij"
      >
        <div
          class="css-h5bbgz"
        >
          <svg
            fill="none"
            height="24"
            viewBox="0 0 24 24"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M10.05 2.52997L4.03002 6.45997C2.10002 7.71997 2.10002 10.54 4.03002 11.8L10.05 15.73C11.13 16.44 12.91 16.44 13.99 15.73L19.98 11.8C21.9 10.54 21.9 7.72997 19.98 6.46997L13.99 2.53997C12.91 1.81997 11.13 1.81997 10.05 2.52997Z"
              stroke="#A7A7A7"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="1.5"
            />
            <path
              d="M5.63012 13.08L5.62012 17.77C5.62012 19.04 6.60012 20.4 7.80012 20.8L10.9901 21.86C11.5401 22.04 12.4501 22.04 13.0101 21.86L16.2001 20.8C17.4001 20.4 18.3801 19.04 18.3801 17.77V13.13"
              stroke="#A7A7A7"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="1.5"
            />
            <path
              d="M21.3999 15V9"
              stroke="#A7A7A7"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="1.5"
            />
          </svg>
        </div>
      </div>
      <div
        class="css-h2ayij"
      >
        <div
          class="css-h5bbgz"
        >
          <svg
            fill="none"
            height="24"
            viewBox="0 0 30 30"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M27.4625 22.9125C27.4625 23.3625 27.3625 23.825 27.15 24.275C26.9375 24.725 26.6625 25.15 26.3 25.55C25.6875 26.225 25.0125 26.7125 24.25 27.025C23.5 27.3375 22.6875 27.5 21.8125 27.5C20.5375 27.5 19.175 27.2 17.7375 26.5875C16.3 25.975 14.8625 25.15 13.4375 24.1125C12 23.0625 10.6375 21.9 9.3375 20.6125C8.05 19.3125 6.8875 17.95 5.85 16.525C4.825 15.1 4 13.675 3.4 12.2625C2.8 10.8375 2.5 9.475 2.5 8.175C2.5 7.325 2.65 6.5125 2.95 5.7625C3.25 5 3.725 4.3 4.3875 3.675C5.1875 2.8875 6.0625 2.5 6.9875 2.5C7.3375 2.5 7.6875 2.575 8 2.725C8.325 2.875 8.6125 3.1 8.8375 3.425L11.7375 7.5125C11.9625 7.825 12.125 8.1125 12.2375 8.3875C12.35 8.65 12.4125 8.9125 12.4125 9.15C12.4125 9.45 12.325 9.75 12.15 10.0375C11.9875 10.325 11.75 10.625 11.45 10.925L10.5 11.9125C10.3625 12.05 10.3 12.2125 10.3 12.4125C10.3 12.5125 10.3125 12.6 10.3375 12.7C10.375 12.8 10.4125 12.875 10.4375 12.95C10.6625 13.3625 11.05 13.9 11.6 14.55C12.1625 15.2 12.7625 15.8625 13.4125 16.525C14.0875 17.1875 14.7375 17.8 15.4 18.3625C16.05 18.9125 16.5875 19.2875 17.0125 19.5125C17.075 19.5375 17.15 19.575 17.2375 19.6125C17.3375 19.65 17.4375 19.6625 17.55 19.6625C17.7625 19.6625 17.925 19.5875 18.0625 19.45L19.0125 18.5125C19.325 18.2 19.625 17.9625 19.9125 17.8125C20.2 17.6375 20.4875 17.55 20.8 17.55C21.0375 17.55 21.2875 17.6 21.5625 17.7125C21.8375 17.825 22.125 17.9875 22.4375 18.2L26.575 21.1375C26.9 21.3625 27.125 21.625 27.2625 21.9375C27.3875 22.25 27.4625 22.5625 27.4625 22.9125Z"
              stroke="#A7A7A7"
              stroke-miterlimit="10"
              stroke-width="1.5"
            />
            <path
              d="M23.125 11.25C23.125 10.5 22.5375 9.35 21.6625 8.4125C20.8625 7.55 19.8 6.875 18.75 6.875"
              stroke="#A7A7A7"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="1.5"
            />
            <path
              d="M27.5 11.25C27.5 6.4125 23.5875 2.5 18.75 2.5"
              stroke="#A7A7A7"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="1.5"
            />
          </svg>
        </div>
      </div>
      <div
        class="css-h2ayij"
      >
        <div
          class="css-h5bbgz"
        >
          <svg
            fill="none"
            height="24"
            viewBox="0 0 30 30"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M21.25 25.625H8.75C5 25.625 2.5 23.75 2.5 19.375V10.625C2.5 6.25 5 4.375 8.75 4.375H21.25C25 4.375 27.5 6.25 27.5 10.625V19.375C27.5 23.75 25 25.625 21.25 25.625Z"
              stroke="#A7A7A7"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-miterlimit="10"
              stroke-width="1.5"
            />
            <path
              d="M21.25 11.25L17.3375 14.375C16.05 15.4 13.9375 15.4 12.65 14.375L8.75 11.25"
              stroke="#A7A7A7"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-miterlimit="10"
              stroke-width="1.5"
            />
          </svg>
        </div>
      </div>
    </div>
    <div
      class="css-1cruzfr"
    >
      <div
        class="css-0"
      >
        <p
          class="chakra-text css-10ircq3"
        >
          About
        </p>
        <p
          class="chakra-text css-1nykzj4"
        />
      </div>
      <div
        class="css-1409d40"
      >
        <div
          class="css-0"
        >
          <p
            class="chakra-text css-10ircq3"
          >
            Age
          </p>
          <p
            class="chakra-text css-evennk"
          />
        </div>
        <div
          class="css-0"
        >
          <p
            class="chakra-text css-10ircq3"
          >
            Gender
          </p>
          <p
            class="chakra-text css-evennk"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
